tests/Unit/Utils/StringsTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Innovation\Tests\Utils;
+
+use Innovation\Tests\BaseTest;
+use Utils\Strings;
+
+class StringsTest extends BaseTest
+{
+    /**
+     * @dataProvider doesStringComeBeforeProvider
+     */
+    public function testDoesStringComeBefore($str1, $str2, $comesBefore)
+    {
+        $this->assertEquals($comesBefore, Strings::doesStringComeBefore($str1, $str2));
+    }
+
+    public function doesStringComeBeforeProvider(): array
+    {
+        return [
+            ['abacus', 'zebra', true], # alpha precedence
+            ['zebra', 'abacus', false], # alpha precedence, reverse case
+            ['1vision', 'abacus', true], # numbers prior to letters
+            ['aaaa', '1abc', false], # numbers prior to letters, reverse case
+            ['aaa', 'abbb', true], # alphabet precedence for second letter
+            ['aaa', 'a11', false], # second numbers over letters
+            ['A Big Card', 'ABC Card', false], # spaces _after_ letters
+            ['Code of Laws', 'Pottery', true], # Real example
```

Comment:
Can you add a real example using accents? e.g. `São Paulo`

Reply:
According to Chris, we should treat accented characters as if they are not accented. So ideally they are treated as equals.

Reply:
So then who goes first if the card is identical but one has accents?

Reply:
There aren't any other cards starting with Sao, and it's possible that's the only accent in the titles.

Can we make sure Sailing comes before São Paulo, and Sao Paulo comes before Satellites? If that works, I think we should be good.